### PR TITLE
Schedule

### PR DESCRIPTION
**Pull request by lucymonie**
relates to #29

### CODE REVIEW

File: coursebook/week-3/schedule.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
+## Schedule
+
+### Monday
+10.00-10.15 - Introduction to gitbook, learning outcomes & project teams
+Start each week with a mission and a goal.
```
Comment (emilyb7) [inline coursebook/week-3/schedule.md]: do we need this line? seems to me that the mission and goal should be the same as the learning outcomes.

File: coursebook/week-3/schedule.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
+## Schedule
+
+### Monday
+10.00-10.15 - Introduction to gitbook, learning outcomes & project teams
+Start each week with a mission and a goal.  
+10.15-13.00 - Workshop 1 (Ewelina & Lucy)
```
Comment (emilyb7) [inline coursebook/week-3/schedule.md]: Do we need our names in here? This info is kind of temporal and maybe not needed in the master reference...

File: coursebook/week-3/schedule.md [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+- Request-response pattern, HTTP, XHR, Postman  
+--LUNCH--
+14.00-16.00 - Workshop 2 (Emily & Nori)
```
Comment (emilyb7) [inline coursebook/week-3/schedule.md]: I would amend the description slightly to mention callbacks first; chaining callbacks using the waterfall pattern should be the stretch goal

File: coursebook/week-3/schedule.md [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+- Implementing waterfall
+- Callbacks, chaining callbacks as a stretch goal
+16.00-18.00 - Business Development / Community Engagement
+
+### Tuesday
+10.00-13.00 - Workshop 3  
+Full-day workshop  
+--LUNCH--  
+14.00-17.15 - Continue full-day workshop  
+17.15-18.00 - Whiteboard Walkthrough + Introduce Group Project
```
Comment (skibinska) [inline coursebook/week-3/schedule.md]: Who is going to do the whiteboard walkthrough?

Comment (emilyb7) [inline coursebook/week-3/schedule.md]: what is the whiteboard walkthrough? i think this is a relic of something we copy and pasted and that it doesn't belong here!

**Comment by emilyb7** [comment]
Have made a new branch for all week-3 related things https://github.com/foundersandcoders/master-reference/pull/126

**Comment by emilyb7** [comment]
@eliascodes thanks for your feedback. I've made some changes in #126 if you want to check again...

**Pull request closed without merging**